plugins/build.gradle.kts

```diff
@@ -27,4 +28,11 @@ subprojects {
             showStackTraces = true
         }
     }
+    tasks.withType(KotlinCompile::class).all {
+        kotlinOptions {
+            freeCompilerArgs = freeCompilerArgs + listOf(
+                "-opt-in=org.jetbrains.dokka.plugability.ExperimentalDokkaPluginApi"
+            )
+        }
+    }
```

Comment:
While convenient, it might hide user annoyances/problems since we wouldn't be dogfooding our own plugin API

If current changes require plugin authors to add too many annoying `OptIn` annotations and they use similar approaches to just suppress all warnings (present and future), it makes the annotation kind of useless

Let's remove it and just opt in place where it's needed? Then we can see better how it'll be used and it would add transparency to our own plugins

Reply:
For a test task, it is the easiest way to add it in the script, otherwise, we would have `OptIn` in every file of tests.

Reply:
It is the easiest for us, but it hides this and future problems. 

If plugin writers have to use `@OptIn` in every single test file, it's annoying, and they will either have to do it for every single class, or will have to suppress it like above, which is not obvious and needs to be googled.

Is there no way to suppress it (opt-in) for the base `testInline` or `BaseAbstractTest` declarations, so that it doesn't require the users to opt in?

Reply:
`DokkaPlugin` is a part of `testInline` signature. 
Changing  `testInline` is not worth it